.github/workflows/build-node-binaries.yaml

```diff
@@ -0,0 +1,45 @@
+# This defines a workflow to build and release a new version of the aptos CLI.
```

Comment:
I have changes pending for this but I want to see the job pass / fail first since it takes so damn long :P